ghenet-Ghenet/week-5/react-giphy/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+//import logo from './logo.svg';
+import './App.css';
+import GiphyList from './components/GiphyList'
+import SearchBar from './components/SearchBar'
+import _ from 'lodash'
+
+const API_KEY = "iqhkltdPbQN8PlLydc4BXLPGzRYycxFj";
+
+class App extends Component {
+  constructor() {
```

Comment:
remember you can remove the constructor now for cleaner code 😄

Reply:
Will do. I'll stop being old school .